grafana-plugin/src/containers/ChannelFilterForm/ChannelFilterForm.tsx

```diff
@@ -33,7 +33,7 @@ const ChannelFilterForm = observer((props: ChannelFilterFormProps) => {
   const [filteringTermJinja2, setFilteringTermJinja2] = useState<string>(
     data ? data.filtering_term_jinja2 : '{payload.foo == "bar"}'
   );
-  const [filteringTermType, setFilteringTermType] = useState<number>(data ? data.filtering_term_type : 1);
+  const [filteringTermType, setFilteringTermType] = useState<FilteringTermType>(data ? data.filtering_term_type : 1);
```

Comment:
@maskin25 can you please help

When I try using enum Value like here:

```
const [filteringTermType, setFilteringTermType] = useState<FilteringTermType>(data ? data.filtering_term_type : FilteringTermType.jinja2);
  
```

frontend fails with 


```
TypeError: undefined is not an object (evaluating 'models_channel_filter_channel_filter_types__WEBPACK_IMPORTED_MODULE_8__.FilteringTermType.jinja2')
```
and compliler shows:
```
Compiled with warnings!
  Attempted import error: 'FilteringTermType' is not exported from 'models/channel_filter/channel_filter.types' (imported as 'FilteringTermType').
  No errors found.

```
<img width="917" alt="Screenshot 2023-02-10 at 21 48 15" src="https://user-images.githubusercontent.com/2262529/218107713-46d8a5da-f3b2-4f88-ae5f-42766c2df622.png">